package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A module for tracking Ethereum token balances over block changes.",
   "main": "dist/index.js",
   "engines": {
-    "node": "^14 || ^16 || >=18"
+    "node": "^16.20.1 || ^18.16.1 || >=20"
```

Comment:
Do we plan on making this version match the module template eventually? (I know this package is out of date and it seems you are progressively updating it, so no worries, just curious at what stage we are at here)